kms-message/test/test_kms_request.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef TEST_KMS_REQUEST_H
+#define TEST_KMS_REQUEST_H
+
+#define ASSERT_CMPSTR(_a, _b) compare_strs (__FUNCTION__, (_a), (_b))
+
+#define ASSERT(stmt)                                                        \
```

Comment:
Should this macro (and others defined in this header) be given a prefix such as `TEST_KMS_` or `KMS_`?